docs/migrations/v7-to-v7_1.md

```diff
@@ -0,0 +1,74 @@
+# Migrating from v7 to v7.1
+
+This guide provides instructions for migrating to version `v7.1.0` of ibc-go.
+
+There are four sections based on the four potential user groups of this document:
+
+- [Migrating from v7 to v7.1](#migrating-from-v7-to-v71)
+  - [Chains](#chains)
+  - [IBC Apps](#ibc-apps)
+  - [Relayers](#relayers)
+  - [IBC Light Clients](#ibc-light-clients)
+
+**Note:** ibc-go supports golang semantic versioning and therefore all imports must be updated on major version releases.
+
+## Chains
+
+In the previous release of ibc-go, the localhost `v1` light client module was deprecated and removed. The ibc-go `v7.1.0` release introduces `v2` of the 09-localhost light client module.
+
+Chain developers must register the 09-localhost `AppModuleBasic` in `app.go`.
```

Comment:
It's not required is it, if you don't want to make use of the localhost?

Reply:
I think it technically is, because ibc core now creates the `ClientState` by default.

Reply:
Hmm, it feels a bit off to have an SDK module be dependent upon the registration of another SDK module..

Reply:
It does actually... maybe localhost types could be registered within ibc core since there already is tighter coupling. It would remove the need for the app module basic in 09-localhost. 

What do you think?

Reply:
opened https://github.com/cosmos/ibc-go/pull/3220